### PR TITLE
Admin music pref changes

### LINKED ISSUE #17361: "Set Admin Volume" defaults to 0 since PR
For some reason the "Set Admin Volume" in preferences is defaulted to 0 rather than 100.
This has confused a lot of admins as they assume they can no longer hear midi/ogg's being played.

It does, after changed, stay at the level it is set to between rounds and log in/out.

Also someone mentioned the "Hear Admin Midi's" toggle button no longer works due to this either (even when at 100) though I have not tested that myself yet.

### PR DESCRIPTION
**Pull request by Shadowlight213**
Removes the admin midi toggle
fixes admin music volume MAYBE.
Save shit is confusing ahhhhh.
Fixes #17361

### CODE REVIEW

**Comment by Jordie0608** [comment]
#17361

File: code/modules/client/preferences_savefile.dm [hunk 1]
```diff
@@ -244,6 +244,8 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 	ghost_orbit 	= sanitize_inlist(ghost_orbit, ghost_orbits, initial(ghost_orbit))
 	ghost_accs		= sanitize_inlist(ghost_accs, ghost_accs_options, GHOST_ACCS_DEFAULT_OPTION)
 	ghost_others	= sanitize_inlist(ghost_others, ghost_others_options, GHOST_OTHERS_DEFAULT_OPTION)
+	if(!S["adminmusicvolume"])		//AHHHH I COULDN'T THINK OF A BETTER WAY TO CHECK IF THIS ISNT SET
```
Comment (optimumtact) [inline code/modules/client/preferences_savefile.dm]: just remove the preference completely you meme

Comment (Shadowlight213) [inline code/modules/client/preferences_savefile.dm]: I guess? I mean not denying headphone users the magic of admin music, while protecting them from super loud ones was kinda the reasoning for the control.

Comment (Dorsisdwarf) [inline code/modules/client/preferences_savefile.dm]: They can _turn their own volume down_

Comment (lzimann) [inline code/modules/client/preferences_savefile.dm]: it wont even change the volume until a new one anyway, so 
\>change volume 
\>next midi is super quiet
\>change back 
\>repeat

**Comment by Shadowlight213** [comment]
Remaking this since It's easier to start from scratch.

**Comment by Shadowlight213** [comment]
#17435 or this.
I'm having people on both sides of the argument here.

**Comment by Shadowlight213** [comment]
Ok. so, I know how to make the volume adjustable for the current sound. The issue is actually calling the var storing it.

File: code/__DEFINES/preferences.dm [hunk 2]
```diff
@@ -1,7 +1,7 @@
 
 //Preference toggles
 #define SOUND_ADMINHELP	1
-#define SOUND_MIDI		2
+#define UNUSED		2
```
Comment (phil235) [inline code/__DEFINES/preferences.dm]: just remove it completely and change the values of the other defines below.

Comment (Shadowlight213) [inline code/__DEFINES/preferences.dm]: wouldn't that mess everyone's prefs up though?

Comment (RemieRichards) [inline code/__DEFINES/preferences.dm]: you'd have to make an update proc to discard the value from savefiles, and then up the savefile version.

**Pull request closed without merging**